lib/bamboo/plug/sent_email_viewer/helper.ex

```diff
@@ -42,6 +42,9 @@ defmodule Bamboo.SentEmailViewerPlug.Helper do
     HTML.html_escape(body || "")
   end
 
+  def format_email_address(emails) when is_list(emails),
```

Comment:
I wonder if there's a better way to handle multiple emails. My hesitation with this approach is that this helper function was meant for a single email. Bamboo uses an internal format for addresses (a tuple of `{name, address}`), so this function is meant to take that and display the email nicely. 

Is it possible to iterate over the email addresses elsewhere and format them one by one with the `format_email_addess/1` below as it stands?

Reply:
After looking into this a bit more, I think this is a good way to handle this. We could iterate on the `sent_email_viewer/index.html` template, if `from` is a list of emails, but I think I like this better than that approach. 

It's also similar to how we the `Bamboo.Formatter` handles lists, though in that case we're generating the 2-tuples rather than formatting for presenting in the email. Still, makes me feel like this is consistent with how we've dealt with this in the past.